Directory.Packages.props

```diff
@@ -58,6 +57,7 @@
     <PackageVersion Include="SimpleExec" Version="12.0.0" />
     <PackageVersion Include="System.IdentityModel.Tokens.Jwt" Version="$(WilsonVersion)" />
     <PackageVersion Include="Microsoft.IdentityModel.Protocols.OpenIdConnect" Version="$(WilsonVersion)" />
+    <PackageVersion Include="System.Net.Http" Version="4.3.4" />
```

Comment:
For tests that run on `net481` (against libraries that have a `netstardard2.0`).